Description: Disable architecture bits detection.
 In Debian, you can use multiarch facilities for choosing which version 
 to run.
Author: Ognyan Kulev <ogi@debian.org>
Forwarded: not-needed 
Last-Update: 2013-04-24 
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- bowtie-1.0.0.orig/Makefile
+++ bowtie-1.0.0/Makefile
@@ -121,6 +121,9 @@
     BITS_FLAG = -m64
 endif
 
+# Debian: Disable selection of 32/64-bits. Use multiarch instead.
+BITS_FLAG =
+
 DEBUG_FLAGS = -O0 -g3 $(BITS_FLAG)
 RELEASE_FLAGS = -O3 $(BITS_FLAG)
 NOASSERT_FLAGS = -DNDEBUG
